Mathlib/Algebra/Lie/Semisimple/Lemmas.lean

```diff
@@ -19,20 +19,16 @@ This file is a home for lemmas about semisimple and reductive Lie algebras.
 
 ## TODO
 
-* It appears that the lemmas in this file may hold for any principal ideal domain (of
-  characteristic zero) instead of a field. The plan would be to define the nilradical of a Lie
-  algebra (currently missing) and use it with
-  `LieModule.exists_nontrivial_weightSpace_of_isNilpotent` instead of using the solvable radical
-  with `LieModule.exists_nontrivial_weightSpace_of_isSolvable`, as below. The conclusion of
-  `LieAlgebra.hasCentralRadical_and_of_isIrreducible_of_isFaithful` would then make a statement
-  that the nilradical vanishes and one would prove elsewhere that vanishing nilradical is
-  equivalent to `HasCentralRadical`.
```

Comment:
I no longer believe this: it might be possible to prove via `LieModule.exists_nontrivial_weightSpace_of_isNilpotent` (rather than Lie's theorem) but one still needs field coeffs and characteristic zero elsewhere in the proof.